packages/data-store/src/migrations/2.simplifyRelations.ts

```diff
@@ -8,22 +8,120 @@ import Debug from 'debug'
  */
 export class SimplifyRelations1447159020002 implements MigrationInterface {
   async up(queryRunner: QueryRunner): Promise<void> {
+    function getTable(givenName: string): Table {
+      const entityMetadatas = queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName);
+      return Table.create(entityMetadatas!, queryRunner.connection.driver);
+    }
     function getTableName(givenName: string): string {
       return (
-        queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName)?.tableName ||
-        givenName
+          queryRunner.connection.entityMetadatas.find((meta) => meta.givenTableName === givenName)?.tableName ||
```

Comment:
Why isn't this simply using getTable to begin with now?

Reply:
As mentioned getTableName is not the real problem in these cases. the change table was the real problem